recipients_app/lib/data/models/dashboard_card.dart

```diff
@@ -88,9 +88,14 @@ class DashboardCard extends StatelessWidget {
       case DashboardCardType.contactNumberEqualsPaymentNumber:
         final recipient = context.read<AuthCubit>().state.recipient!;
 
+        final organization = context.read<AuthCubit>().state.organization!;
```

Comment:
`organization!` can lead to NullPointerExceptions probably. Should we make it without `!` as `AccountPage` accepts `Organization?` ?

Reply:
fixed